Makefile

```diff
@@ -72,4 +72,7 @@ e2e:
 e2e-prod:
 	FOCUS="\[Real\]" ./hack/e2e.sh
 
+e2e-scaleoutin:
```

Comment:
for now, fix out the tags such that this would run on every PR, rather than breaking open another Makefile target and associated release work

Reply:
please can we standardise on the terminology of scale *up/down*, not *out/in*?

Reply:
> please can we standardise on the terminology of scale up/down, not out/in?

+1

Reply:
FWIW, this terminology is backwards in the kubernetes world.